README.md

```diff
@@ -12,3 +12,6 @@ Installation Instructions (rough notes)
 2. install dependencies in requirements.txt
 3. install npm dependencies
 4. Get an API Key with Movie Database API and make file local_settings.py with Global Variable saving it
+
+To populate the database, you also need to install Django-Extensions:
```

Comment:
This should be updated to what we talked about; updating the above-instructions to do `pip install -r requirements.txt` instead, as that lists the dependency.

Reply:
Yes, and at some point I'll have to make this readme a bit better in general